packages/web/src/scss/components/Avatar/README.md

```diff
@@ -56,16 +56,31 @@ The content of the `Avatar` component can be an image, an icon, or a text string
 
 ### Icon
 
-Add an icon with correct size.
+Add an icon. The icon size is automatically set based on the Avatar size.
+You can override the icon size by setting the `width` and `height` attributes on the SVG,
+or by using CSS custom properties.
```

Comment:
Again, I think it's fine to keep the instructions about overriding only in the section dedicated to icon.

```suggestion
```